kornia/augmentation/augmentation.py

```diff
@@ -1106,11 +1106,25 @@ class Normalize(AugmentationBase2D):
     """
 
     def __init__(
-        self, mean: torch.Tensor, std: torch.Tensor,
+        self, mean: Union[torch.Tensor, Tuple[float, ...], float],
+        std: Union[torch.Tensor, Tuple[float, ...], float],
+        num_channels: Optional[int] = 3,
```

Comment:
What if tensor gives 4 while num_channels got only 3. A bit confusing. I would remove this arg.

Probably use torch.expand(mean, input.shape[1]) when len(mean.shape) == 0.

Reply:
the point is to avoid any extra computations when function is called.

Reply:
I wouldn't encourage this if it is conflicting with other params. 

Meanwhile, torch.expand will not consume much and do not require the user to know their input in the first place.

I am okay with both solutions, either solve the confusion or with torch.expand.

Reply:
ok will use torch expand then